src/jwplayer/players/shared/CloseButton/CloseButton.tsx

```diff
@@ -16,9 +17,17 @@ interface CloseButtonProps {
 
 const CloseButton: React.FC<CloseButtonProps> = ({ className, dismiss, style, iconColor, iconSize }) => {
 	const { player } = useContext(PlayerContext);
+	const communicationService = getCommunicationService();
+
 	const onClickClose = (event) => {
 		event.stopPropagation();
 		jwPlayerPlaybackTracker({ event_name: 'video_player_close' });
+		communicationService.dispatch({
+			type: '[Video] Close button clicked',
+			payload: {
+				player: 'JW player',
```

Comment:
Is it possible to use the player context to dynamically determine the player's name instead of hardcoding it?